easybib/recipes/role-phpapp.rb

```diff
@@ -16,6 +16,4 @@
 
 include_recipe 'tideways'
 
-if is_aws
```

Comment:
@till: was there a specific reason for this change or is it an artifact of fiddling around?